src/main/java/models/Game.java

```diff
@@ -70,8 +71,14 @@ private void dealerTurn() {
             endHand("Dealer Wins");
         } else {
             endHand("You win!");
-            player.win();
+            if(player.getCount() == 21 && player.getHand().size() == 2){
```

Comment:
Remove the player.getHand().size()
The player can win with a "blackjack" i.e., 21, if they have more than 2 cards in their hand.